crates/burn-import/onnx-tests/tests/onnx_tests.rs

```diff
@@ -459,6 +460,24 @@ mod tests {
         assert!(expected_sum_2d.approx_eq(output_sum_2d, (1.0e-4, 2)));
     }
 
+    #[test]
+    fn slice() {
+        let model: slice::Model<Backend> = slice::Model::default();
+        let device = Default::default();
+
+        let input = Tensor::<Backend, 2>::from_floats(
+            [
+                [1., 2., 3., 4., 5., 6., 7., 8., 9., 10.],
+                [1., 2., 3., 4., 5., 6., 7., 8., 9., 10.],
```

Comment:
Maybe change the values of the second dim just to make it clear what data is sliced with this test 😅

Reply:
😃 right, yeah